mod/metrics/set/abstract/metric_answer/flags.rb

```diff
@@ -25,6 +25,26 @@ def commented?
     end
   end
 
+  def calculated_flag
+    return "" unless card.calculated?
+    calculated_flag_icon
+  end
+
+  def calculated_flag_icon
+    return overridden_flag_icon if card.calculation_overridden?
```

Comment:
let's go with `researched_value?` instead of `calculation_overridden?`.  This will handle cases where there is no calculated value to override (because the variable metrics aren't populated).